tests/tensor_ops/test_getitem.py

```diff
@@ -48,6 +47,22 @@ def adv_index_bool_wrap(*arrs):
     return hnp.arrays(shape=arrs[0].shape, dtype=bool)
 
 
+def arb_index_wrap(*arrs):
+    return arbitrary_indices(arrs[0].shape)
+
+
+def test_index_empty():
+    a = mg.Tensor([])
```

Comment:
Ugh sorry. That was my fault.

```suggestion
    a = Tensor([])
```